src/sidebar.jsx

```diff
@@ -94,7 +114,20 @@ export const SidebarPanelDetails = ({ selected, path, setPath, showHidden, setSh
                         <DescriptionListTerm>{_("Size")}</DescriptionListTerm>
                         <DescriptionListDescription>{cockpit.format("$0 $1", cockpit.format_bytes(selected.size), selected.size < 1000 ? "B" : "")}</DescriptionListDescription>
                     </DescriptionListGroup>}
+                    <DescriptionListGroup id="description-list-owner-permissions">
+                        <DescriptionListTerm>{_("Owner permissions")}</DescriptionListTerm>
+                        <DescriptionListDescription>{getPermissions(selected.permissions.slice(0, 1))}</DescriptionListDescription>
```

Comment:
You just need here the first character. You can get it with selected.permissions[0] which is more readable